modules/ingest-service-impl/src/main/java/org/opencastproject/ingest/impl/IngestServiceImpl.java

```diff
@@ -249,7 +247,7 @@
           .build();
 
   /** The default is to overwrite series catalog on ingest */
-  protected boolean defaultIsOverWriteSeries = true;
+  protected boolean defaultIsOverWriteSeries = false;
 
```

Comment:
Update or remove the old comment.

Reply:
I went with a static constant I could use here as well